low-level/sensors/impl/generic/node_hw.py

```diff
@@ -436,6 +445,12 @@ def run(self):
 
                     self._update_list_file()
                     self._notify_NodeDataMsgHandler()
+                    try:
+                        self._check_faulty_resource_status()
+                    except Exception as e:
+                        logger.error("_check_faulty_resource_status, error %s "
```

Comment:
"Direct check for faulty resources recovery failed with error %s"

Reply:
Done